src/response.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * Utility module that converts sequelize models into SensorThings format.
+ */
+
+const FIELDS_TO_FILTER = [
+  'id',
+  'createdAt',
+  'updatedAt'
+];
+
+const formatItem = function formatItem (item) {
+  const itemData = item.dataValues;
+  const resourceName = item.$modelOptions.name.plural;
+  let formatedItem = {
+    '@iot.id': itemData.id,
+    '@iot.selfLink': '/' + resourceName + '(' + itemData.id + ')'
+  };
+
+  Object.keys(itemData).forEach(key => {
+    if (FIELDS_TO_FILTER.indexOf(key) === -1) {
+      formatedItem[key] = itemData[key];
+    }
+  });
+
+  return formatedItem;
+}
+
+const generate = function generate (resource) {
+  if (resource && Array.isArray(resource)) {
+    let response = {};
+    response['@iot.count'] = resource.length;
+    response.value = [];
+    resource.forEach(item => {
+      response.value.push(formatItem(item));
+    });
+
+    return response;
+  }
+
+  return formatItem(resource);
+}
+
+module.exports = {
+  generate
+};
```

Comment:
I am adding a couple changes to this file at https://github.com/mozilla-sensorweb/sensorthings/pull/7/files#diff-d0f5704ae0738a7bd1f54aff42ddcb41R1 so I'll save the comments here :)